### PR TITLE
Basic Routing Completed

### PR DESCRIPTION
**Pull request by Ranapriyanshi**
Added basic routes link to the index page.

### CODE REVIEW

**Comment by 2005balaji** [comment]
@Ranapriyanshi  pls leave a description!

File: apps/web/components/Navbar.vue [hunk 1]
```diff
@@ -2,16 +2,17 @@
...
       <!-- <h1>Navbar</h1> -->
       <div class="w-2/6">
-        <img :src="logo" alt="" class="h-24"/>
+        <nuxt-link to="/"><img :src="logo" alt="" class="h-24"/></nuxt-link>
+        
       </div>
       <div class="flex justify-evenly items-center p-15 w-3/6">
-        <h1 class="p-15">About</h1>
-        <h1>Blogs</h1>
+        <h1 class="p-15"><nuxt-link to="/about">About</nuxt-link></h1>
```
Comment (2005balaji) [inline apps/web/components/Navbar.vue:9]: pls format the code to view at ease!

**Review by Arpit1204 — APPROVED** [review]
LGTM

File: apps/web/components/Navbar.vue (hunk 1 above)
Comment (Ranapriyanshi) [reply apps/web/components/Navbar.vue:9]: Formatted the code.👍

**Pull request merged**